weaviate/collections/classes/config_vectorizers.py

```diff
@@ -745,10 +781,58 @@ def text2vec_palm(
             vectorizeClassName=vectorize_collection_name,
         )
 
+    @staticmethod
+    def multi2vec_palm(
+        *,
+        location: str,
+        project_id: str,
```

Comment:
Like above, should these go above the `*,` since they're required? Actually, is it in case they stop being required so we reorder them as necessary?